### ISSUE TITLE
Rules not enabled by default

### ISSUE THREAD

**Issue by sidprak** [terraform-linters/tflint#1117]
## Terraform Code

```hcl
variable "unused" {
}
```

## Output

Works with --only:
```console
% tflint --only terraform_unused_declarations .
1 issue(s) found:

Warning: variable "unused" is declared but not used (terraform_unused_declarations)

  on main.tf line 1:
   1: variable "unused" {

Reference: https://github.com/terraform-linters/tflint/blob/v0.28.0/docs/rules/terraform_unused_declarations.md
```

Without any options, it passes:
```console
% tflint .; echo $?
0
```

Is this expected behavior? Is there a way to enable all rules by default without listing every rule by name?

## Version

Example:

```console
% tflint -v
TFLint version 0.28.0
```

**Comment by bendrucker** [comment]
https://github.com/terraform-linters/tflint/tree/master/docs/rules

There is currently no flag for enabling all rules, you're best off opting in via the config file.
